packages/foreman/rubygem-sassc/rubygem-sassc.spec

```diff
@@ -25,10 +28,14 @@ BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
+BuildRequires: libsass
+Requires: libsass
+# We explicitly point it to libsass.so.0 in %build
+Requires: libsass.so.0()
```

Comment:
Fedora has `libsass.so.1` so this is a bit of an ugly hack. Maybe EL8 has `libsass.so.1` as well but by making this explicit, repoclosure would pick it up. It was inspired by rubygem-sassc in Fedora.